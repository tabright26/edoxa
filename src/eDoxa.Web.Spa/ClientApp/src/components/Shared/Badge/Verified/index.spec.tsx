import React from "react";
import renderer from "react-test-renderer";
import { Verified } from ".";

it("renders without crashing", () => {
  const tree = renderer
    .create(<Verified verified={false} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
