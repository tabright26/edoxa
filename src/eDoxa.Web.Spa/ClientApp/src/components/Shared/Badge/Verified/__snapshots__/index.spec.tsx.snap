// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<span
  className="px-2 null badge badge-secondary badge-pill"
>
  <span>
    Unverified
  </span>
</span>
`;
